extensions/wikia/EditPageLayout/css/EditPageLayout.scss

```diff
@@ -599,3 +599,9 @@ body {
 		@include breakpoints-large-typography;
 	}
 }
+
+#infoboxPreview {
```

Comment:
If I use a class selector, I have to apply `!important` to some of the styles. Since this is only applying to this specific element, doesn't it make more sense to use id?